deployment/roles/injector/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+- hosts: 
```

Comment:
this file looks like a playbook instead of task file. Besides, it has duplicated code already coded in ` deployment/injector.yml`

Reply:
Thanks for spotting that. I've cleaned up the tasks in the playbook. They will be executed by the role.